runtime/interpreter/interpreter.go

```diff
@@ -425,6 +435,16 @@ func WithOnResourceOwnerChangeHandler(handler OnResourceOwnerChangeFunc) Option
 	}
 }
 
+// WithOnMeterComputationFuncHandler returns an interpreter option which sets
+// the given function as the meter computation handler.
+//
+func WithOnMeterComputationFuncHandler(handler OnMeterComputationFunc) Option {
```

Comment:
Needs to be propagated in `NewSubInterpreter`

Reply:
nice catch

Reply:
updated at 7f84226